Pipfile

```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 pytest = "*"
+"boto3" = "*"
```

Comment:
Is there a reason why there's quotation marks here? We should remove them for consistency if they're not needed.